docker-load.md

```diff
@@ -84,7 +91,77 @@ You should see output like:
 
 ```
 REPOSITORY                         TAG                 IMAGE ID            CREATED             SIZE
-ccdl/training_rnaseq               2019-chicago        22f2b4f05051        3 days ago          5.28GB
+ccdl/training_rnaseq               2019-czi        22f2b4f05051        3 days ago          5.28GB
```

Comment:
Can you remove this image id in a way that indicates it needs to be updated by us at a later point?

Reply:
Yes. I will do that.